src/vector_of_array.jl

```diff
@@ -64,16 +64,10 @@ end
 struct AllObserved
 end
 
-Base.@pure __parameterless_type(T) = Base.typename(T).wrapper
-
-@generated function issymbollike(x)
-    x <: Union{Symbol, AllObserved} && return quote true end
-    ss = ["Operation", "Variable", "Sym", "Num", "Term"]
-    s = string(Symbol(__parameterless_type(x)))
-    bool = any(x -> occursin(x, s), ss) 
-    quote 
-        $bool
-    end
+function issymbollike(x)
+    x isa Union{Symbol, AllObserved} && return true
```

Comment:
Just dispatch in Symbolics in an extension

Reply:
what should the behavior be in pre 1.9?